source/leuven/services/openondemand.rst

```diff
@@ -436,16 +442,15 @@ You can do this by using the ``lib`` argument for both the ``install.packages``
   You will also notice that you cannot use the same way of navigating after this.
   Another solution is to click the three dots on the right (...) and enter your path.
 - The 'Tools-Install packages' interface does not allow you to select any other path than the default in your ``$VSC_HOME``.
-  It is recommended to use the ``install.packages`` function instead.
+  It is recommended to use the ``install.packages()`` function instead.
 - RStudioServer will by default store the RStudio cache in ``$VSC_HOME/.local/share/rstudio``.
   This cache can get very large, and cause you to exceed the quota of your home directory.
-  To avoid this, you can redirect this cache to your data directory by setting ``$XDG_DATA_HOME``
-  variables in your ``~/.bashrc``.
+  To avoid this, you can redirect this cache to your scratch directory by setting the ``$XDG_DATA_HOME``
```

Comment:
In section 7.3.2, we are recommending the users to set `XDG_DATA_HOME` to somewhere in their data folder! Clearly, this will be confusing for the users.
Each of these two choices have pros and cons; hence, I recommend to make a decision within our team and fix this in this page coherently.

Reply:
I am personally leaning towards advising scratch, especially for R users as the RStudio cache can get very large and may unknowingly occupy a large part of someone's data storage. We can definitely discuss this with the team.

Reply:
Maxime proposed a nice compromise. To stick with `XDG_DATA_HOME=$VSC_DATA/.local/share`. But also for RStudio specifically, so inform the users about some default settings to not by default save the RSession when RStudio is closed. This to avoid the RStudio Cache from unknowingly filling up a good chunk of the user's data directory.
What do you think @moravveji ?

Reply:
This is a good idea. If we manage to change the RStudio default behavior (not saving a closed session), then we can stick to forwarding the caches to VSC_DATA using `XDG_DATA_HOME` consistently across this page.
But, the question is whether we can change the RStudio default behavior.

I looked up `/apps/leuven/rocky8/icelake/2022b/software/RStudio-Server/2022.07.2+576-foss-2022b-Java-11-R-4.2.2/R/ServerOptions.R`, and there I find:

```R
# set save defaults for high-performance (only if the administrator
# or user hasn't set them explicitly already)

if (is.null(getOption("save.defaults")))
   options(save.defaults=list(ascii=FALSE, compress=FALSE))

if (is.null(getOption("save.image.defaults")))
   options(save.image.defaults=list(ascii=FALSE, safe=TRUE, compress=FALSE))
```
But, this is too vague to me; the comment line is also ambiguous and I wonder if this is what we are looking after. Furthermore, the passed values are mainly `FALSE` which do not give extra room to turn off auto-saving (if that is what these two lines mean).

So, any other idea to turn off auto saving of sessions?

Reply:
In RStudio, it is 'easy' to override the defaults and suppress loading of previous sessions upon startup. It is just under 'Tools -> Global Options -> General -> RSessions'.

However, I looked up the source code for the equivalent of these, so that we can perhaps modify these defaults in `Options.R` and/or `ServerOptions.R`. But, so far, I have not spotted from where they actually come from. Can you perhaps ask this from the Posit support?